src/.editorconfig

```diff
@@ -473,3 +473,12 @@ dotnet_diagnostic.CA1051.severity = suggestion
 # copyright header
 file_header_template = Copyright (c) Microsoft Corporation.\nLicensed under the MIT License.
 
+# Verify settings
+[*.{received,verified}.{txt,xml,json}]
+charset = "utf-8-bom"
+end_of_line = lf
```

Comment:
this is fine for mac/linux , and would require Windows folks to update to crlf

Reply:
Not really. Windows can do `lf` line endings, it just defaults to `crlf`. Admittedly it's unlikely to be noticed as you don't open most files matching this pattern, except when doing new tests and the `.gitattributes` will also take care of line ending changes.

Reply:
I change this locally on my Windows box to `crlf`. When set to `lf`, running `dotnet format` changes all my files with the crlf line endings.
I also have `git config core.autocrlf` set to **true**.

Reply:
I've run into a lot of annoying messages about line ending too, if this can help fix that, I am all for it :)

Reply:
@aaronburtle - alas no, it won't. I think to fix it we need to do something with the `.gitattributes` and `*.cs` files, but it'll appear as a whole file change for like _every_ file, so I would do it in a standalone PR 😅